app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        all_rule_done = True

```

Comment:
You don't need this variable to make it work - simply return result on last line if none of the `if` conditions trigger an exception

Reply:
Remove all_rule_done variable and refactor code below so it works without it